charts/gatekeeper/crds/config-customresourcedefinition.yaml

```diff
@@ -24,10 +24,14 @@ spec:
       description: Config is the Schema for the configs API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation
```

Comment:
You can leave out the unrelated generated manifests from this PR (config,constraints, etc..)

Reply:
This was actually part of a previous PR, added to the branch to make it compile. It will disappear once the previous PRs are merged.